### DISCUSSION TITLE
Companion App not work

### DISCUSSION THREAD

**Discussion by tonivag** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#217] [Q&A] [answered]
Hi, i upgrade to last version and from browser work find, but in the App i have an error:
Missing configuration: map_image or map_camera

Why on the PC with browser work, and from mobile with App i have a error?

Thank you
Hi

**Comment by PiotrMachowski** [comment, marked as answer]
You have cached v1 version of the card in companion app. Try cleaning cache or changing URL in resources by adding `?v=v2.0.5`

**Reply by tonivag** [reply]
Ok, perfect!
Now work Fine!

Hi

**Comment by ficwiktor** [comment]
Had the same problem. For some reason, map came back after I turned on Xiaomi app (I had no reason to for a while - I keep all my automations in HA).

Before I turned on an app I tried:
- redownload vacuum-map-card
- turn on viomi integration (for some reason HA turned it off "by config entry") 
-- it didn't turn off after reset
- check on token&ip with token extractor
- clearing cache

Clearing cache didn't work for me. Our problems are resolved, but I want to post it anyway for other users.

**Reply by PiotrMachowski** [reply]
> Had the same problem

What do you mean?

**Reply by ficwiktor** [reply]
Map didn't show, but all buttons worked. I had an info "Missing configuration: map_image or map_camera".
Turning on an app seemed to repair the problem.
Before that I did all the steps listed above.

**Reply by PiotrMachowski** [reply]
Can you show a screenshot?

**Reply by ficwiktor** [reply]
I don't have a screenshot. But it looked like first screenshot in this 
https://lightrun.com/answers/piotrmachowski-home-assistant-custom-components-xiaomi-cloud-map-extractor-map-extractor-dont-show-map

System logs filtered by "error":
![image](https://user-images.githubusercontent.com/52634139/215543478-34a3d93e-ca2d-4681-851d-68891e060ab4.png)

**Reply by PiotrMachowski** [reply]
Well, in this case you have a completely different problem, related to map extractor, not map card

**Reply by ficwiktor** [reply]
Ok, thanks
